src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,7 +104,11 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setMaxWidth(70);
+			if (getTable().getColumnExt(0).getHeaderRenderer()==null) {// If there isn't a header renderer then setMaxWidth
+				Component c = getTable().getTableHeader().getDefaultRenderer().getTableCellRendererComponent(null, getTable().getColumnExt(0).getHeaderValue(), false, false, 0, 0);
+				getTable().getColumnExt(0).setMaxWidth(c.getMaximumSize().width);
```

Comment:
Ok I'm confused yesterday you said "(although we should check and use the column's header renderer, if it has one)". Which I tried to accommodate here, if null then set otherwise let the specified renderer do it's thing. 

Now you're saying that I should always use these two statements except when I shouldn't....are we stuck in a loop?